### PR TITLE
Adding route line example activity with inline comments for documentation

### PR DESCRIPTION
**Pull request by cafesilencio**
### Description
Adding route line example activity with inline comments for documentation

### CODE REVIEW

File: test-app/src/main/java/com/mapbox/navigation/examples/internal/MapboxRouteLineActivity.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package com.mapbox.navigation.examples.core;
+package com.mapbox.navigation.examples.internal;
```
[the remark this answers is not in the document]
Comment (cafesilencio) [reply test-app/src/main/java/com/mapbox/navigation/examples/internal/MapboxRouteLineActivity.java:1]: It's not for internal testing it's for my daily development work.

File: test-app/src/main/java/com/mapbox/navigation/examples/core/MapboxRouteLineAndArrowActivity.kt [hunk 2]
```diff
@@ -0,0 +1,517 @@
...
+                        MapboxRouteLineAndArrowActivity::class.java.simpleName,
+                        "Error loading map: " + mapLoadError.name
+                    )
+                }
+            }
+        )
+    }
+
+    override fun onMapLongClick(point: Point): Boolean {
+        vibrate()
```
[the remark this answers is not in the document]
Comment (cafesilencio) [reply test-app/src/main/java/com/mapbox/navigation/examples/core/MapboxRouteLineAndArrowActivity.kt:372]: You're talking about the vibrate()?

[the remark this answers is not in the document]
Comment (abhishek1508) [reply test-app/src/main/java/com/mapbox/navigation/examples/core/MapboxRouteLineAndArrowActivity.kt:372]: Yes

[the remark this answers is not in the document]
Comment (cafesilencio) [reply test-app/src/main/java/com/mapbox/navigation/examples/core/MapboxRouteLineAndArrowActivity.kt:391]: removed

File: test-app/src/main/java/com/mapbox/navigation/examples/MainActivity.kt [hunk 3]
```diff
@@ -0,0 +1,163 @@
+package com.mapbox.navigation.examples
```
Comment (abhishek1508) [inline test-app/src/main/java/com/mapbox/navigation/examples/MainActivity.kt:1]: This file needs to be deleted as it is in `test-app`
@cafesilencio

Comment (cafesilencio) [reply test-app/src/main/java/com/mapbox/navigation/examples/MainActivity.kt]: deleted

**Review by abhishek1508 — APPROVED** [review]

**Pull request merged**